array.js

```diff
@@ -0,0 +1,8 @@
+function array_largest(arr){
```

Comment:
Using array.sort function will cause some bugs. Please fix your code.

Reply:
Fixed!

Reply:
Code looks good. Can you please add some documentation?